pkg/kobject/kobject.go

```diff
@@ -107,3 +108,15 @@ type Ports struct {
 	HostIP        string
 	Protocol      api.Protocol
 }
+
+// Volumes holds the volume struct of container
+type Volumes struct {
+	SvcName    string
+	MountPath  string
+	VFrom      string
+	VolumeName string
+	Host       string
+	Container  string
+	Mode       string
+	PVCName    string
+}
```

Comment:
@cdrage I think this is needed for doing the whole volumesFrom thing reliably, since these will be resolved once to save computation later.